modules/ROOT/pages/automated-policy-apply.adoc

```diff
@@ -0,0 +1,35 @@
+= Applying Automated Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+. In *API Manager*, select *Automated Policies* from the left navigation menu.
+. Click *Apply new automated policy*.
+. Select the provided policy that you want to configure as a provided policy. +
+You can configure any provided policies for Mule 4 as an automated policy. +
+See the xref:policies-ootb-landing-page.adoc[list of available Provided Policies] to learn how to configure each one of them, or the xref:policy-mule3-creating-custom-policy.adoc[Creating a Custom Policy] article to learn how to create one.
++
+When configuring an automated policy, at the bottom of the policy configuration page, there is a *Rule of Applications* sub-section. This dictates the set of runtimes that will be affected by the policy:
+
+* *Apply to all supported runtimes (4.1.1 and above)*. +
+All API proxies based on Mule 4.1.1 and later will be affected by the policy.
+* *Apply to a specific set of runtimes*. +
```

Comment:
I think it should be a good idea to clarify that:
- The input for the **To** is not required; and moreover, leaving it blank means that the set of runtimes covered will go from the value placed in **From** and above (No upper limit). Example:
  - _From_: 4.1.5 _To_: (This will cover from 4.1.5 onwards)

- The inputs **From** and **To** are [semver](https://semver.org/) compliant. Meaning the users can the ranges of runtimes like:
  - _From_: 4.1.1 _To_: 4.1.x (This will cover for all runtime versions of 4.1)
  - _From_: 4.2.x _To_: 4.2.9 (This will cover from the minimum version of 4.2, probably _4.2.0_, until 4.2.9 included)